### PR TITLE
use `pathlib` to handle a `WEBBPSF_PATH` with double slashes

### PR DESCRIPTION
**Pull request by zacharyburnett**
to handle cases such as
```
E           OSError: WEBBPSF_PATH (/home/runner/work/_temp/data//webbpsf-data/) is not a valid directory path!
```
https://github.com/spacetelescope/romancal/actions/runs/9974996761/job/27563753075#step:10:305

### CODE REVIEW

**Review by BradleySappington — APPROVED** [review]
In pathlib we trust!

**Comment by BradleySappington** [comment]
@zacharyburnett , code approved for pathlib handling, however please fix the pep8speaks suggestion (and tests working)

**Comment by zacharyburnett** [comment]
> @zacharyburnett , code approved for pathlib handling, however please fix the pep8speaks suggestion (and tests working)

the tests are blocked by #885 (and https://github.com/spacetelescope/webbpsf/actions/workflows/download_data.yml needs to be rerun after that merge)

**Pull request merged**
